src/render.rs

```diff
@@ -75,11 +80,41 @@ impl<'a> MarkdownRenderer<'a> {
             },
             ..ComrakOptions::default()
         };
-        let rendered = comrak::markdown_to_html(text, &options);
+
+        let arena = Arena::new();
+        let root = parse_document(&arena, text, &options);
+
+        // Tweak annotations of code blocks.
+        iter_nodes(root, &|node| {
+            if let NodeValue::CodeBlock(ref mut ncb) = node.data.borrow_mut().value {
+                let mut orig_annot = String::from_utf8(ncb.info.to_vec()).unwrap();
```

Comment:
Let's avoid the unwrap here. If the value isn't utf-8, we could either skip the `,` check, or replace the info with an empty string.

Reply:
That's reasonable, addressed via 7cba38b.